mlir/python/CMakeLists.txt

```diff
@@ -710,11 +710,10 @@ if(NOT LLVM_ENABLE_IDE)
     COMPONENT mlir-python-sources
   )
 endif()
-
-################################################################################
+# ###############################################################################
 # The fully assembled package of modules.
 # This must come last.
-################################################################################
+# ###############################################################################
```

Comment:
Is this like auto-formatting changes here?

Reply:
Yeah, looks like it. I reverted the auto-formatting changes.
Thanks for the review!